pkgs/development/libraries/libkml/default.nix

```diff
@@ -11,13 +11,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libkml";
-  version = "1.3.0";
+  version = "unstable-2017-01-15"; # Last release is 1.3.0
```

Comment:
I suspect only https://github.com/libkml/libkml/commit/9163d288dfae64139563190dbf09f82ec28e8559 is actually needed, but applying that patch directly fails for some thunks. The other option would be hand-crafting a .patch file

Reply:
```suggestion
  version = "unstable-2017-01-15";
```